### ISSUE TITLE
error trying to reach service: Address is not allowed, error code: 500 . Lens metrics not available

### ISSUE THREAD

**Issue by NagabhushanaYR** [lensapp/lens#3609]
Hi ,  I am not able to see lens metrics.  Below details are captured in the logs and I am not sure if this is something to do with RBAC Or if any specific access / configurations are missed.

Logs:
[31merror[39m: [Metrics]: metrics not available {"error":{"name":"StatusCodeError","statusCode":500,"message":"500 - {\"kind\":\"Status\",\"apiVersion\":\"v1\",\"metadata\":{},\"status\":\"Failure\",\"message\":\"error trying to reach service: Address is not allowed\",\"code\":500}","error":{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"error trying to reach service: Address is not allowed","code":500},"options":{"timeout":0,"resolveWithFullResponse":false,"json":true,"qs":{"query":"sum(node_filesystem_size_bytes{kubernetes_node=~\"ip-xx-xx-xx-xxx.ec2.internal|ip-xx-xx-xx-xxx.ec2.internal|ip-10-xx-xx-xxx.ec2.internal|ip-xx-xx-xx-xxx.ec2.internal|ip-xx-xx-xx-xx.ec2.internal\", mountpoint=\"/\"} - node_filesystem_avail_bytes{kubernetes_node=~\"ip-xx-xx-xx-xxx.ec2.internal|ip-xx-xx-xx-xxx.ec2.internal|ip-xx-xx-xx-xxx.ec2.internal|ip-xx-xx-xx-xxx.ec2.internal|ip-xx-xx-xx-xxx.ec2.internal\", mountpoint=\"/\"}) by (kubernetes_node)","start":"1629084660","end":"1629088260","step":"60","kubernetes_namespace":""},"headers":{"Host":"xxxxxxxxx.localhost:64536"},"uri":"http://localhost:64536/api-kube/api/v1/namespaces/lens-metrics/services/prometheus:80/proxy/api/v1/query_range","simple":true,"transform2xxOnly":false},"response":{"statusCode":500,"body":{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"error trying to reach service: Address is not allowed","code":500},"headers":{"access-control-allow-origin":"*","audit-id":"c1947fd4-64ac-4ada-b67a-75bd25759f92","cache-control":"no-cache, private","content-length":"146","content-type":"application/json","date":"Mon, 16 Aug 2021 04:31:34 GMT","connection":"close"},"request":{"uri":{"protocol":"http:","slashes":true,"auth":null,"host":"localhost:64536","port":"64536","hostname":"localhost","hash":null,"search":"?query=sum%28node_filesystem_size_bytes%7Bkubernetes_node%3D~

OS Version: Windows 10
Lens version:  4.2.4

I have reinstalled lens metrics from settings multiple times but no luck and getting the same message in lens UI and same error message in logs. 

I am seeing the same issue with latest lens ( 5.x.x ) versions also.  
![image](https://user-images.githubusercontent.com/77423814/129513170-cc485d27-03e3-41a7-8081-8a4dff294071.png)

**Comment by Nokel81** [comment]
Are you trying to deploy metrics onto an EKS cluster? 

Are you sure that you have access to the machine that the metrics pod is/will be running? For example: is the VPC configuration accessible?

**Comment by NagabhushanaYR** [comment]
@Nokel81 :  Yes.  I am deploying to EKS and from my kube.config file, I am able to access the cluster nodes, run kubectl commands to access all cluster workloads without any issues.  I am able to portforward through kubectl commands and access len's prometheus ( localhost) without any issues.  But I can definitely cross check if you want me to verify any specific access level / run commands etc.

**Comment by NagabhushanaYR** [comment]
@Nokel81 :  When I try to deploy standalone prometheus along with kube-prom-stack then I can able to access metrics in lens.  I guess some setting might be missing when I install prometheus operator.   
I would like   add  - '--web.listen-address=0.0.0.0:9090' flag to args in prometheus,  Can you guide me how to achieve this.
